surface/src/3rdparty/opennurbs/opennurbs_object.cpp

```diff
@@ -840,7 +840,7 @@ void ON_ClassId::ConstructorHelper( const char* sClassName,
   {
     for ( ON_ClassId* p = m_p0; p; p = p->m_pNext )
     {
-      if ( !p->m_pBaseClassId && p->m_sBaseClassName ) {
+      if ( !p->m_pBaseClassId ) {
```

Comment:
Can you replace with these lines, please: https://github.com/mcneel/opennurbs/blob/8.x/opennurbs_object.cpp#L806-L808 As far as I understand, these make sure that the string `p->m_sBaseClassName` ends with a null character/terminating character, but is not an empty string.